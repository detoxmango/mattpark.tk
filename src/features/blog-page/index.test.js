import React from 'react'
import renderer from 'react-test-renderer'

import BlogPage from './index'


describe("Blog", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<BlogPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})