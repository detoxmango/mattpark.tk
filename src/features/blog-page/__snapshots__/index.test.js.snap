// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog renders correctly 1`] = `
Array [
  <h1
    className="MuiTypography-root MuiTypography-h1"
  >
    My Blog
  </h1>,
  <h2
    className="MuiTypography-root MuiTypography-h2"
  >
    Under construction..
  </h2>,
]
`;
